invenio_app_ils/cli.py

```diff
@@ -6,7 +6,8 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """CLI for Invenio App ILS."""
-
+import random
+import time
 from datetime import datetime, timedelta
 from random import randint
```

Comment:
Either `import random` and use `random.randit()` and `random.choice()` or `from random import randint, choice`

Reply:
it is changed anyway already by other PR